src/components/layout/CommandPaletteModal.tsx

```diff
@@ -27,35 +28,44 @@ interface Props {
   selectedCategory: string;
   isSearching: boolean;
   setIsSearching: (isSearching: boolean) => void;
+  onClose: () => void;
 }
 
-const CommandPaletteModal = (props: Props) => {
-  const {
-    isOpen,
-    result,
-    defaultResult,
-    goto,
-    searchInputRef,
-    searchContainerRef,
-    setQuery,
-    query,
-    setSelectedIndex,
-    selectedIndex,
-    setSelectedCategory,
-    selectedCategory,
-    isSearching,
-    setIsSearching,
-  } = props;
-
+export function CommandPaletteModal({
+  isOpen,
+  result,
+  defaultResult,
+  goto,
+  searchInputRef,
+  searchContainerRef,
+  setQuery,
+  query,
+  setSelectedIndex,
+  selectedIndex,
+  setSelectedCategory,
+  selectedCategory,
+  isSearching,
+  setIsSearching,
+  onClose,
+}: Props) {
   const { theme } = useThemeContext();
 
   const selectedItem = result.grouped[selectedCategory]?.[selectedIndex];
   const PreviewIcon = useMemo(
     () => getCategoryIcon(selectedItem?.category || ''),
     [selectedItem?.category],
   );
+
   return (
     <>
+      {isOpen && (
+        <Head>
+          <meta
+            name="viewport"
+            content="width=device-width, initial-scale=1, maximum-scale=1"
+          />
+        </Head>
+      )}
```

Comment:
We do not need to toggle the prop to add the meta tag. Just add the tag to: https://github.com/dwarvesf/memo.d.foundation/blob/29f4157bc1961341f4895f0e47c09193c172a6ae/src/pages/_document.tsx#L23